sources/packages/api/src/route-controllers/user/model/user.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from "class-validator";
+
+export default class UserSyncInfoDto {
+  @IsNotEmpty()
+  firstName: string;
+  @IsNotEmpty()
+  lastName: string;
+  @IsNotEmpty()
+  email: string;
+
```

Comment:
This blank line is supposed to be removed by the formatter. Is the formatter working properly?